js/models/messages.js

```diff
@@ -93,6 +93,8 @@
       return {
         timestamp: new Date().getTime(),
         attachments: [],
+        pow: false,
+        sent: false,
```

Comment:
Do you you still need these?

Reply:
Yep, without them messages won't have the field so fetching from the db on these values will fail